lightcycle-integration-test/src/main/java/com/soundcloud/lightcycle/sample/basic/callback/ActivityCallback.java

```diff
@@ -0,0 +1,6 @@
+package com.soundcloud.lightcycle.sample.basic.callback;
+
+public enum ActivityCallback {
```

Comment:
I tried to match Android's Activity Lifecycle document, but I also think it'll be better to match ActivityLightCycle so it won't cause any confusion.